cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -1361,6 +1361,8 @@ public void unsubscribe(UnsubscribedListener listener) {
 
       CleverPushHttpClient.post("/subscription/unsubscribe", jsonBody,
           new UnsubscribeResponseHandler(this, listener).getResponseHandler());
+    } else {
+      clearSubscriptionData();
```

Comment:
Can you explain this change?

Reply:
I am doing the same as the IOS code for unsubscribe

https://github.com/cleverpush/cleverpush-ios-sdk/blob/404a5aa405aedca7b29be83d27c62f3449032e1b/CleverPush/Source/CleverPushInstance.m#L1032

Reply:
@gbhrdt Should I remove this?